src/Factories/FunFunFactory.php

```diff
@@ -1202,9 +1203,17 @@ public function getMembershipApplicationFormTemplate(): TwigTemplate {
 			'countries' => $this->getCountries(),
 			'addressValidationPatterns' => $validation->address,
 			'dateOfBirthValidationPattern' => $validation->dateOfBirth,
+			'incentives' => $this->getChoiceFactory()->getDefaultIncentives()
```

Comment:
Do we still need this top-level value here?  I don't see any access from applicationdata, IFAIK it's only in `initialformValues`.

Reply:
Oh, no we don't I left it in by accident